Tests/SwiftGodotMacrosTests/MacroGodotTests.swift

```diff
@@ -0,0 +1,175 @@
+		// Note when editing: Xcode loves to change all indentation to be consistent as either tabs or spaces, but the macro expansion produces a mix.
+		// I had to set Settings->Text Editing->Tab Key to "Inserts a Tab Character" in order to resolve this.
+		assertMacroExpansion(
+			"""
+			@Godot class Hi: Node {
+				func hi() {
+				}
+			}
+			""",
+            expandedSource: """
+            class Hi: Node {
+            	func hi() {
+            	}
+            
+                required init(nativeHandle _: UnsafeRawPointer) {
+                	fatalError("init(nativeHandle:) called, it is a sign that something is wrong, as these objects should not be re-hydrated")
+                }
+
+                required init() {
+                	Hi._initClass ()
+                	super.init ()
+                }
+
+                static func _initClass () {
+                    let className = StringName("Hi")
+                    let classInfo = ClassInfo<Hi> (name: className)
+                }
+            }
+            """,
+			macros: testMacros
+		)
+	}
+	
+	func testGodotMacroWithCallableFunc() {
+		// Note when editing: Xcode loves to change all indentation to be consistent as either tabs or spaces, but the macro expansion produces a mix.
+		// I had to set Settings->Text Editing->Tab Key to "Inserts a Tab Character" in order to resolve this.
+		assertMacroExpansion(
+			"""
+			@Godot class Castro: Node {
+				@Callable func deleteEpisode() {
+				}
+			}
+			""",
+            expandedSource: """
+            class Castro: Node {
+            	func deleteEpisode() {
+            	}
+            
+            	func _mproxy_deleteEpisode (args: [Variant]) -> Variant? {
+            		deleteEpisode ()
+            		return nil
+            	}
+            
+                required init(nativeHandle _: UnsafeRawPointer) {
+                	fatalError("init(nativeHandle:) called, it is a sign that something is wrong, as these objects should not be re-hydrated")
+                }
+
+                required init() {
+                	Castro._initClass ()
+                	super.init ()
+                }
+
+                static func _initClass () {
+                    let className = StringName("Castro")
+                    let classInfo = ClassInfo<Castro> (name: className)
+                	classInfo.registerMethod(name: StringName("deleteEpisode"), flags: .default, returnValue: nil, arguments: [], function: Castro._mproxy_deleteEpisode)
+                }
+            }
+            """,
+			macros: testMacros
+		)
+	}
+	
+	func testExportGodotMacro() {
+		assertMacroExpansion(
+			"""
+			@Godot class Hi: Node {
+				@Export var goodName: String = "Supertop"
+			}
+			""",
+			expandedSource:
+            """
+            class Hi: Node {
+            	var goodName: String = "Supertop"
+            
+            	func _mproxy_set_goodName (args: [Variant]) -> Variant? {
+            		/*    goodName = String(args [0])!*/
```

Comment:
Not changed in this PR, but surprising that this expands to a `/* */` block — is this a bug @migueldeicaza?

Reply:
Yes, this was some leftover debugging code, I will remove it.